modules/nw-ptp-configuring-linuxptp-services-as-ordinary-clock.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="configuring-linuxptp-services-as-ordinary-clock_{context}"]
+= Configuring linuxptp services as Ordinary Clock
+
+The PTP Operator adds the `PtpConfig.ptp.openshift.io` custom resource definition (CRD) to {product-title}.
+You can configure the `linuxptp` services (`ptp4l`, `phc2sys`) by creating a `PtpConfig` custom resource (CR) object.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator.
+
+.Procedure
+
+. Create the following `PtpConfig` CR, and then save the YAML in the `ordinary-clock-ptp-config.yaml` file.
++
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: ordinary-clock-ptp-config <1>
+  namespace: openshift-ptp
+spec:
+  profile: <2>
+  - name: "profile1" <3>
+    interface: "ens787f1" <4>
+    ptp4lOpts: "-s -2" <5>
+    ptp4lConf: ... <6>
```

Comment:
Same comment here. I think we should have a full complete example.

Reply:
Yes elipsis ia a wrong value. You need to add the full `ptp4lConf` or you can remove `ptp4lConf` field

Reply:
@evgenLevin Could you provide an example about `ptp4lConf` for the ordinary clock as well?

Reply:
Can we leave it without `ptp4lConf`?
https://github.com/openshift/ptp-operator/blob/master/config/samples/ptp_v1_ptpconfig.yaml
https://docs.openshift.com/container-platform/4.3/networking/multiple_networks/configuring-ptp.html